docs/source/data_format.rst

```diff
@@ -101,22 +84,13 @@ reader (`rispy <https://pypi.org/project/rispy/>`__). Successful import/export
 depends on a proper data set structure. The complete list of accepted fields and 
 default mapping can be found on the `rispy GitHub page <https://github.com/MrTango/rispy>`_.
 
+The labels ``ASReview_relevant``, ``ASReview_irrelevant``, and
+``ASReview_not_seen`` are stored with the N1 (Notes) tag, and can be
+re-imported into ASReview LAB. The behavior of the labels is different for
```

Comment:
Not part of this PR, isn't it?

Reply:
Yes, because due to the changes in the validation mode, lines 65-87 are outdated and had to be updated. I decided to remove the information in this part of the documentation to avoid overlap with other parts to keep the docs as simple as possible.